README.md

```diff
@@ -1,12 +1,13 @@
 - [OpenSearch Plugins](#opensearch-plugins)
-    - [Naming Conventions](#naming-conventions)
-    - [Managing OpenSearch Plugins](#managing-opensearch-plugins)
-    - [Building Plugins with OpenSearch](#building-plugins-with-opensearch)
-    - [Upgrading Plugins to work with OpenSearch](#upgrading-plugins-to-work-with-opensearch)
-    - [Installing Plugins](#installing-plugins)
-    - [Plugin Release Notes](#plugin-release-notes)
-    - [Headers](#headers)
+  - [Naming Conventions](#naming-conventions)
+  - [Managing OpenSearch Plugins](#managing-opensearch-plugins)
+  - [Building Plugins with OpenSearch](#building-plugins-with-opensearch)
+  - [Upgrading Plugins to work with OpenSearch](#upgrading-plugins-to-work-with-opensearch)
+  - [Installing Plugins](#installing-plugins)
+  - [Plugin Release Notes](#plugin-release-notes)
+  - [Headers](#headers)
 - [Contributing](#contributing)
+- [Standards](#standards)
```

Comment:
This should go under OpenSearch Plugins like a subsection. Just like any other topic here.
The main header is for something which talks about this repository itself.

Reply:
I have updated this in the README.